app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/controller.js");
+
+app.use(express.json());
```

Comment:
we're not using this line yet, please take it out until the code requires you to put it in. Think about what it's doing and at what endpoint you expect to need it